src/app/dataset-view/additional-components/settings-component/settings.component.sass

```diff
@@ -1,139 +1,140 @@
 @import src/assets/styles/var
 p
-    margin: 0
+	margin: 0
```

Comment:
Could you please check `settings.component.sass`, for some reason all indents are changed from spaces to tabs, it's not consistent with the project.

Reply:
Fixed